### PR TITLE
MP-6-I made a questionnaire

### PR DESCRIPTION
**Pull request by winLaymon**

### CODE REVIEW

File: src/app/user-form/user-form.component.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.bvn{
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:1]: rename classes

File: src/app/user-form/user-form.component.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  text-align: center;
+}
+.bll{
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:4]: rename classes

File: src/app/user-form/user-form.component.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  text-align: center;
+  height: 39px;
+  width: 193px;
+  border-radius: 19px;
+  border: 1px solid black;
+  background-color: bisque;
+  margin-top: 20px;
+}
+.bnn{
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:13]: rename classes

File: src/app/user-form/user-form.component.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  text-align: center;
+  height: 39px;
+  width: 193px;
+  border-radius: 19px;
+  border: 1px solid black;
+  background-color: bisque;
+  margin-top: 20px;
+}
+
+.bnm{
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:23]: rename classes

File: src/app/user-form/user-form.component.css [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  height: 42px;
+  width: 100%;
+  border-radius: 20px;
+  border: 1px solid black;
+  background-color: bisque;
+  margin-top: 20px;
+}
+
+
+.vbnm{
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:34]: rename classes

File: src/app/user-form/user-form.component.css [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+  margin: 0 auto;/*отступы 1-высота.2-ширина*/
+  width: 200px ;/*ширина*/
+  margin-top: 90px;
+}
+
+.cvbnm{
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:40]: rename classes

File: src/app/user-form/user-form.component.css [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+  border: none;
+  padding: 10px 25px;
+  background: #ff9900;
+  cursor: pointer;
+  text-transform: uppercase;
+  font-weight: bold;
+  color: white;
+  margin-top: 20px;
+}
+button[name="run_script"]:hover {
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:63]: add class

File: src/app/user-form/user-form.component.html [hunk 8]
```diff
@@ -1 +1,26 @@
...
+    <option value="2">Санкт-Петербург</option>
+    <option value="3">Новосибирск</option>
+    <option value="4">Киев</option>
+    <option value="5">Харьков</option>
+    <option value="6">Львов</option>
+    <option value="7">Минск</option>
+    <option value="8">Бобруйск</option>
+    <option value="9">Гомель</option>
+  </select>
+  <button  type="button" name="run_script" >Отправить</button>
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.html:25]: add class

File: src/app/user-form/user-form.component.css [hunk 9]
```diff
@@ -0,0 +1,65 @@
+.bvn{
+  text-align: center;
+}
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:3]: add an empty structure

Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:3]: add empty structure after }

File: src/app/user-form/user-form.component.css [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+.bll{
+  text-align: center;
+  height: 39px;
+  width: 193px;
+  border-radius: 19px;
+  border: 1px solid black;
+  background-color: bisque;
+  margin-top: 20px;
+}
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:12]: add empty structure after }

File: src/app/user-form/user-form.component.css [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+  border-radius: 19px;
+  border: none;
+  padding: 10px 25px;
+  background: #ff9900;
+  cursor: pointer;
+  text-transform: uppercase;
+  font-weight: bold;
+  color: white;
+  margin-top: 20px;
+}
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:62]: add empty structure after }

File: src/app/user-form/user-form.component.css [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+.bnm{
+  text-align: center;
+  height: 42px;
+  width: 100%;
+  border-radius: 20px;
+  border: 1px solid black;
+  background-color: bisque;
+  margin-top: 20px;
+}
+
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:32]: remove empty line

File: src/app/user-form/user-form.component.css [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
```
Comment (Anastasiamatiek) [inline src/app/user-form/user-form.component.css:33]: remove empty line

**Review by Anastasiamatiek — APPROVED** [review]

File: src/app/user-form/user-form.component.css (hunk 7 above)
Comment (DeylBlack) [reply src/app/user-form/user-form.component.css:63]: there are not need add class here. this code related to just button tag

File: src/app/user-form/user-form.component.css (hunk 1 above)
Comment (DeylBlack) [reply src/app/user-form/user-form.component.css:1]: I agree with that. We can rename this class to .user-form-header

File: src/app/user-form/user-form.component.css (hunk 2 above)
Comment (DeylBlack) [reply src/app/user-form/user-form.component.css:4]: same here. rename class to .phone-number

**Review by DeylBlack — APPROVED** [review]

**Pull request merged**